scripts/helper.js

```diff
@@ -29,7 +29,7 @@ const endpoints = {
   local: 'http://127.0.0.1:8888',
   kylin: 'http://kylin.fn.eosbixin.com',
   telosTestnet: 'https://test.hypha.earth',
-  telosMainnet: 'https://node.hypha.earth'
+  telosMainnet: 'https://api.telosfoundation.io'
```

Comment:
@sebastianmontero I had timeouts trying to deploy proposals so had to use another node - not sure why our node times out on deploy, or what makes it time out vs. other contracts deployed?